.github/workflows/main.yml

```diff
@@ -74,6 +74,8 @@ jobs:
       run: docker exec test_container /PrairieLearn/docker/lint_js.sh
     - name: Run the HTML linter
       run: docker exec test_container /PrairieLearn/docker/lint_html.sh
+    - name: Run the links linter
+      run: docker exec test_container /PrairieLearn/docker/lint_links.sh
```

Comment:
@mwest1066 this fails in CI because we don't mount `graders`/`workspaces`/`images`/`.gitattributes` into the container. Are you opposed to just running this specific step on the host instead of in the container?

Reply:
I'm fine to run it outside the container. We are generally only running things inside the container to make sure we've got the right version of node and everything else.